frontend/src/lib/components/HistoryList/HistoryList.scss

```diff
@@ -0,0 +1,21 @@
+@import '~/vars.scss';
+
+.history-list {
+    .history-list-row {
+        display: flex;
+        margin-top: $default_spacing;
+
+        &:first-of-type {
+            margin-top: $default_spacing / 2;
```

Comment:
Considering that the empty state has no margin, shouldn't the first element of the list have none too?

Reply:
The empty (and loading state) having no margin is the error here I think

Although https://github.com/PostHog/posthog/pull/8927#issuecomment-1062262904 the empty state is going to be removed

Reply:
(the margin at the top is from the design)